### PR TITLE
feat: [PIE-5815]: Remove feature flag CD_TRIGGERS_REFACTOR for Service v2 triggers

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by pankajsingh-harness** [comment]
retrigger titlecheck

File: src/modules/72-triggers/pages/triggers/views/TriggersList.tsx [hunk 1]
```diff
@@ -189,7 +173,7 @@ export default function TriggersList(props: TriggersListPropsInterface & GitQuer
 
     return (
       <AddDrawer
-        addDrawerMap={getCategoryItems(getString, hideArtifactManifestSelection)}
+        addDrawerMap={getCategoryItems(getString, false)}
```
Comment (AshwinBhatkal-Harness) [inline src/modules/72-triggers/pages/triggers/views/TriggersList.tsx:176]: We can remove the second argument here. We don't need it anymore. This was added only for service v2

**Review by vkbansal-harness — APPROVED** [review]

File: src/modules/72-triggers/components/Triggers/TriggerWidget.tsx [hunk 2]
```diff
@@ -29,11 +27,12 @@ export function TriggerWidget<T>({
   isNewTrigger,
   triggerData
 }: TriggerWidgetProps<T>): JSX.Element {
-  const isTriggersRefactor = useFeatureFlag(FeatureFlag.CD_TRIGGERS_REFACTOR)
+  const { NG_SVC_ENV_REDESIGN = false, CD_TRIGGERS_REFACTOR = false } = useFeatureFlags()
```
Comment (AshwinBhatkal-Harness) [inline src/modules/72-triggers/components/Triggers/TriggerWidget.tsx:30]: The default value is boolean, we don't need to assign a false default value to this

File: src/modules/72-triggers/pages/triggers/views/TriggersList.tsx (hunk 1 above)
Comment (pankajsingh-harness) [reply src/modules/72-triggers/pages/triggers/views/TriggersList.tsx:176]: this is still needed to show coming soon option

File: src/modules/72-triggers/components/Triggers/TriggerWidget.tsx (hunk 2 above)
Comment (pankajsingh-harness) [reply src/modules/72-triggers/components/Triggers/TriggerWidget.tsx:30]: Done

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by vkbansal-harness — APPROVED** [review]

**Pull request merged**